catalogue/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -29,7 +29,8 @@ import {
   CatalogueResultsList,
   Concept as ConceptType,
   Image as ImageType,
-  ResultType,
+  toWorkBasic,
+  WorkBasic,
   Work as WorkType,
```

Comment:
Just double checking it's worth keeping this type?

Reply:
Yeah, I still want it in this file.

This is used on concept pages for concepts that *aren't* genres. In that case, we want to simulate getting an empty response from the works API for "works that use this type" – then we can transform the API response into the WorkBasic type we use in the page.

You don't absolutely need it, but I think it makes the code clearer and avoids an unnecessary `unknown` type.